flowmachine/tests/test_displacement.py

```diff
@@ -19,7 +19,7 @@
         ("stddev", 182.885304966881, 218.698764480622),
         ("avg", 370.301009034846, 389.155343931983),
         ("sum", 11109.0302710454, 11285.5049740275),
-        ("variance", 33447034.7728291, 47829149.5853505),
+        ("variance", 33447.0347728291, 47829.1495853505),
```

Comment:
Looks like `Displacement` was previously giving wrong answers for `statistic="variance"` when `unit="km"`, but would have been correct when `unit="m"`. Can we add a `statistic="variance"` case to `test_returns_expected_result_for_unit_m` to ensure it's correct for both choices of unit?